lexicons/app/bsky/feed/getPollAnswers.json

```diff
@@ -0,0 +1,83 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.feed.getPollAnswers",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Get poll answers for a given poll which reference a post.",
```

Comment:
That's a downside to the implementation I have put together so far. I considered instead creating an individual record called a Poll, which would then be associated with a Post, which could mitigate this. The issue I had with that is then you have this object that can only really ever have a relationship to a post, or can otherwise just be orphaned, and I figured that wouldn't be very useful.

I'm open to changing this though!